demisto_sdk/commands/common/hook_validations/pre_process_rules.py

```diff
@@ -0,0 +1,117 @@
+import os
+from distutils.version import LooseVersion
+
+import click
+from demisto_sdk.commands.common.constants import \
+    PRE_PROCESS_RULES_BUILT_IN_FIELDS
+from demisto_sdk.commands.common.errors import Errors
+from demisto_sdk.commands.common.hook_validations.content_entity_validator import \
+    ContentEntityValidator
+from demisto_sdk.commands.common.tools import (get_all_incident_and_indicator_fields_from_id_set)
+from demisto_sdk.commands.common.update_id_set import BUILT_IN_FIELDS
+
+FROM_VERSION_PRE_PROCESS_RULES = '6.5.0'
+
+
+class PreProcessRulesValidator(ContentEntityValidator):
+    def __init__(self, structure_validator=True, ignored_errors=False, print_as_warnings=False,
+                 json_file_path=None, **kwargs):
+        super().__init__(structure_validator, ignored_errors, print_as_warnings,
+                         json_file_path=json_file_path, **kwargs)
+        self.from_version = self.current_file.get('fromServerVersion')
+        self.to_version = self.current_file.get('toServerVersion')
+
+    def is_valid_file(self, validate_rn=True):
```

Comment:
@roysagi should this function be called in the `validate_manager`?

Reply:
@tkatzir - we do not need both `is_valid_file` and `is_valid_pre_process_rules` - I think we needed that in the past for format support but new format just runs validate's flow.